src/entities/MovieDetails/MovieDetails.jsx

```diff
@@ -0,0 +1,31 @@
+import "./MovieDetails.css";
+
+export const MovieDetails = ({
```

Comment:
let's pass one property (e.g.`movieInfo`) that contains all these props instead of passing 6 props

Reply:
done